src/Parser/MarksmanshipHunter/Modules/FocusChart/MaxFocus.js

```diff
@@ -0,0 +1,11 @@
+import Module from 'Parser/Core/Module';
+
+class MaxFocus extends Module {
```

Comment:
I think you should be able to combine this and the `ActiveFocusWasted` into 1 module that tracks focus.